crates/valence_command/src/manager.rs

```diff
@@ -0,0 +1,87 @@
+use bevy_app::{App, Plugin, PreUpdate, Update};
+use bevy_ecs::entity::Entity;
+use bevy_ecs::prelude::{Added, Commands, IntoSystemConfigs, Or, Query, Res};
+use bevy_ecs::query::Changed;
+use valence_server::client::{Client, SpawnClientsSet};
+use valence_server::protocol::packets::play::CommandTreeS2c;
+use valence_server::protocol::WritePacket;
+
+use crate::command_graph::CommandGraph;
+use crate::command_scopes::CommandScopes;
+use crate::{CommandRegistry, CommandScopeRegistry};
+
+pub struct CommandManagerPlugin;
```

Comment:
Is the main plugin for the whole lib? I would prefer the name `CommandPlugin` for consistency.